docs/api.apib

```diff
@@ -88,6 +113,52 @@ Creates a new image based on a JSON object containing the following metadata.
                 "required": [ "label","published_at", "url" ]
             }
 
+All fields are required, 
+images with missing metadata fields will return an error
+
++ Request (application/json)
```

Comment:
This is actually ok (from a documentation point of view) - and actually [recomended](http://dredd.org/en/latest/how-to-guides.html#multiple-requests-and-responses).

The only note is that it might be better to add the label on the `request`stanza:

```apiary
+ Request Error: Images with missing metadata fields will return an error (application/json)
```

This will fix the doc output, and make clear to the reader that it is an error:

![image](https://user-images.githubusercontent.com/25080094/36602889-ee2ad1e4-1897-11e8-9f33-810786ec59c9.png)